<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Journals</string>
  </property>
  <property name="toolTip">
   <string>Enter a searc term. Can include wild cards</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>371</width>
      <height>394</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="2">
      <widget class="QPushButton" name="pushButtonFind">
       <property name="toolTip">
        <string>Enter a search term in the searchbox and click Find</string>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEditPages">
       <property name="toolTip">
        <string>Format: Start page - End page. Ex 31-45</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBoxVolume">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QComboBox" name="comboBoxSearchField">
       <property name="toolTip">
        <string>Select the field to be searched</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinBoxIssue">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEditJournal"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Issue</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Pages</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="toolTip">
        <string>Enter search tem. You can use wildcards</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="toolTip">
        <string>Add the data in the fields above, then click Add</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditAuthor">
       <property name="toolTip">
        <string>Format: First name, Last name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Journal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxYear">
       <property name="maximum">
        <number>2050</number>
       </property>
       <property name="value">
        <number>2020</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="toolTip">
        <string>To remove row, first select it by clicking the number to the left of it, then click Remove</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QPushButton" name="pushButtonClear">
       <property name="toolTip">
        <string>Clear the search results</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
